meta/SaiSerialize.cpp

```diff
@@ -90,24 +90,27 @@ static sai_status_t transfer_list(
 {
     SWSS_LOG_ENTER();
 
-    if (countOnly || dst_element.count == 0)
```

Comment:
why this logic is changed ?

Reply:
Will extract this change in a different PR